util/omrutil/CMakeLists.txt

```diff
@@ -122,6 +123,7 @@ list(APPEND OBJECTS
 	detectVMDirectory.c
 	gettimebase.c
 	j9memclr.c
+        omrcrc32.c
```

Comment:
change space indentation to tab

Reply:
Thanks for the comment. This change is incorporated in the recent push.

Reply:
> change space indentation to tab

Thanks for the comment. I have this incorporated in the recent push.